src/components/DocWrapper/DocList/DocList.test.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import {render, screen} from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+
+import DocList, {DocListProps} from './index';
+
+describe('DocList', () => {
+    const props: DocListProps = {
+    className: 'test-class',
+    variant: 'ul',
+    };
+
+  it('renders without crashing', () => {
+    render(<DocList {...props}>Hello World</DocList>);
+  });
+
+  it('renders with proper default className', () => {
+    render(<DocList {...props}>Hello World</DocList>);
+    const list = screen.getByTestId(`DocList--${props.variant}`);
```

Comment:
I think you can create a `const testIdUl = 'DocList--ul` and `const testIdOl = 'DocList--ol` and reuse them all over